ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java

```diff
@@ -217,6 +217,22 @@ else if (qaaa.split("=")[0].equals( tab+tab+tab+"country_name" ) ) {
                                             qaaa = scnr.nextLine();
                                             qaaa = qaaa.split("=")[1];
                                             qaaa = qaaa.substring(1,qaaa.length()-1);
+                                            if (qaaa.equals("CIVILWAR_FACTION_NAME")) {
+                                                int civilWarFlag = 0;
+                                                while (civilWarFlag == 0) {
+                                                    qaaa = scnr.nextLine();
+                                                    qaaa = qaaa.replace(tab,"");
+                                                    if (qaaa.split("=")[0].equals("name")) {
+                                                        qaaa = qaaa.split("=")[1];
+                                                        qaaa = qaaa.substring(1,qaaa.length()-1);
+                                                        if (qaaa.equals("CIVILWAR_FACTION_NAME")) {
```

Comment:
As Codacy note, this if statement is empty. Reverse the logic to make it simpler (unless this is a placeholder for logic in the if):

```
if (!qaaa.equals("CIVILWAR_FACTION_NAME")) {
  civilWarFlag = 1;
}
```

Reply:
The logic has been reversed. Is PR#37 now good to merge?